alerting/src/utils/capture-utils.ts

```diff
@@ -0,0 +1,103 @@
+import puppeteer, { Browser, Page, BoundingBox } from 'puppeteer';
+
+interface CropRegion {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+}
+
+interface ScreenshotOptions {
+  url: string;
+  crop?: CropRegion;
+  screenshotTargetSelector?: string;
+  elementsToHide?: string[];
+}
+
+const DEFAULT_CROP: CropRegion = {
+  x: 0,
+  y: 0,
+  width: 1920,
+  height: 1080,
+};
+const DEFAULT_TARGET = '.maplibregl-canvas';
+
+async function captureScreenshotFromUrl(options: ScreenshotOptions): Promise<string> {
```

Comment:
Add a docstring that includes an example on how to use itAdd a docstring that includes an example on how to use it